### ISSUE TITLE
loadtest: load test the new registry

### ISSUE THREAD

**Issue by matifali** [coder/internal#261]
We have recently moved the Coder registry https://registry.coder.com to a new refactored one. We would like to load test it.

It could be as simple as including some modules in the load-testing template. 

```tf
module "jetbrains_gateway" {
  source         = "registry.coder.com/modules/jetbrains-gateway/coder"
  version        = "1.0.25"
  agent_id       = coder_agent.example.id
  agent_name     = "example"
  folder         = "/home/coder/example"
  jetbrains_ides = ["CL", "GO", "IU", "PY", "WS"]
  default        = "GO"
}
```

cc: @bcpeinhardt to coordinate with @f0ssel when this happens. We may need to keep an eye on the registry.
Ideally, we should have monitoring #182 and #137 before we do this.

**Comment by bcpeinhardt** [comment]
@Parkreiner and I did some "load testing" prior to moving prod over. We ran something like 3000 requests over the course of a couple minutes and nothing was too bothered or too slow. 
As it stands now I'm not convinced we need anything like regular load testing, as things are running pretty smoothly under the current production load, but maybe a load testing script in the repo for running against dev if we make changes we think may impact performance? Could be useful for detecting any introduction of memory leaks etc.
My instinct is with the existing architecture (cloud run deployed to us-central serving statically generated files), distance and the possibility of a cold start will effect performance more than anything else, and a regularly running load test would cost as much as the service itself.

**Comment by matifali** [comment]
I agree that we do not need this now but should include it in our regular load testing @f0ssel is working on. Moving it to low priority now.
